src/components/HeaderAuth.jsx

```diff
@@ -50,10 +45,9 @@ HeaderAuth.defaultProps = {
   initialised: false,
 };
 
-function mapStateToProps(state) {  // Listens for changes in the Redux Store
-  return {
-    user: state.login.user,
-    initialised: state.login.initialised,
-  };
-}
+const mapStateToProps = (state) => ({
```

Comment:
Just curious, in case I was missing a change in function scope or something. 

@simoneduca the function is executed automatically in the Module pattern. That's why there are extra parentheses at the end of the declaration.